events-server/schema/observer/observer.proto

```diff
@@ -0,0 +1,90 @@
+// Copyright 2019 Authors of Hubble
```

Comment:
we should be able to use the generated protobuf api code from cilium, like hubble does here:

https://github.com/cilium/hubble/blob/eb30e651c7a119fb676825f4431bed89c32a75f7/cmd/observe/observe.go#L26

this way we are not copying the protobuf definitions.

Reply:
done